packages/block-editor/src/components/block-settings-menu-controls/README.md

```diff
@@ -16,3 +16,12 @@ function ReusableBlocksMenuItems() {
 	);
 }
 ```
+
+## Props
+
+### `context`
+
+-   **Type:** `String`
+-   **Default:** `undefined`
+
+A string representing the context where the component is being included. This can be used to conditionalise certain beheaviours.
```

Comment:
```suggestion
A string representing the context where the component is being included. This can be used to conditionalize certain behaviors.
```
We should probably use US spelling here.